pkg/services/apiserver/options/storage.go

```diff
@@ -4,20 +4,23 @@ import (
 	"fmt"
 	"net"
 
-	grafanarest "github.com/grafana/grafana/pkg/apiserver/rest"
 	"github.com/spf13/pflag"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/apiserver/pkg/server/options"
+
+	grafanarest "github.com/grafana/grafana/pkg/apiserver/rest"
 )
 
 type StorageType string
 
 const (
-	StorageTypeFile        StorageType = "file"
-	StorageTypeEtcd        StorageType = "etcd"
-	StorageTypeLegacy      StorageType = "legacy"
-	StorageTypeUnified     StorageType = "unified"
-	StorageTypeUnifiedGrpc StorageType = "unified-grpc"
+	StorageTypeFile            StorageType = "file"
+	StorageTypeEtcd            StorageType = "etcd"
+	StorageTypeLegacy          StorageType = "legacy"
+	StorageTypeUnified         StorageType = "unified"
+	StorageTypeUnifiedGrpc     StorageType = "unified-grpc"
+	StorageTypeUnifiedNext     StorageType = "unified-next"
+	StorageTypeUnifiedNextGrpc StorageType = "unified-next-grpc"
```

Comment:
Once we are comfortable with this interface running in cloud, we will remove these two options